packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -265,104 +203,96 @@ const RelationInput = ({
+          label={label}
+          labelAction={labelAction}
+          disabled={disabled}
+          placeholder={placeholder}
+          hasMoreItems={searchResults.hasNextPage}
+          loading={searchResults.isLoading}
+          onOpenChange={handleMenuOpen}
+          noOptionsMessage={() => noRelationsMessage}
+          loadingMessage={loadingMessage}
+          onLoadMore={() => {
+            onSearchNextPage();
+          }}
+          textValue={value}
+          filterValue={filterValue}
+          onChange={(relationId) => {
+            setValue('');
+            setFilterValue('');
+            onRelationConnect(options.find((opt) => opt.id === relationId));
+            updatedRelationsWith.current = 'onChange';
+          }}
+          onTextValueChange={(text) => {
+            setValue(text);
+          }}
+          onInputChange={(event) => {
+            setFilterValue(event.currentTarget.value);
+            onSearch(event.currentTarget.value);
+          }}
+        >
+          {options.map((opt) => {
+            return <Option key={opt.id} {...opt} />;
+          })}
+        </Combobox>
+        {shouldDisplayLoadMoreButton && (
+          <TextButton
+            disabled={paginatedRelations.isLoading || paginatedRelations.isFetchingNextPage}
+            onClick={handleLoadMore}
+            loading={paginatedRelations.isLoading || paginatedRelations.isFetchingNextPage}
+            startIcon={<Refresh />}
+          >
+            {labelLoadMore}
+          </TextButton>
         )}
-      </Relation>
-    </Field>
+      </Flex>
+      {relations.length > 0 && (
+        <RelationList overflow={overflow}>
+          <VisuallyHidden id={ariaDescriptionId}>{listAriaDescription}</VisuallyHidden>
+          <VisuallyHidden aria-live="assertive">{liveText}</VisuallyHidden>
+          <List
+            height={dynamicListHeight}
+            ref={listRef}
+            outerRef={outerListRef}
+            itemCount={totalNumberOfRelations}
+            itemSize={RELATION_ITEM_HEIGHT + RELATION_GUTTER}
+            itemData={{
+              name,
+              ariaDescribedBy: ariaDescriptionId,
+              canDrag: canReorder,
+              disabled,
+              handleCancel: onCancel,
+              handleDropItem: onDropItem,
+              handleGrabItem: onGrabItem,
+              iconButtonAriaLabel,
+              labelDisconnectRelation,
+              onRelationDisconnect,
+              publicationStateTranslations,
+              relations,
+              updatePositionOfRelation: handleUpdatePositionOfRelation,
+            }}
+            itemKey={(index) => `${relations[index].mainField}_${relations[index].id}`}
+            innerElementType="ol"
+          >
+            {ListItem}
+          </List>
+        </RelationList>
+      )}
+      {(description || error) && (
+        <Box paddingTop={2}>
+          <FieldHint />
+          <FieldError />
+        </Box>
+      )}
```

Comment:
Do we still need this or does the combobox render it (asking because you are passing it in already)?

Reply:
I think we can remove it, yeah 🤔